### ISSUE TITLE
图片无法转换

### ISSUE THREAD

**Issue by hgmsq** [microsoft/markitdown#114]
测试结果图片不能转换到md文档里面

**Comment by SigireddyBalasai** [comment]
make sure that you are using latest version

**Comment by hgmsq** [comment]
Today, I tested directly using the pip command to install dependency packages.

**Comment by SigireddyBalasai** [comment]
> Today, I tested directly using the pip command to install dependency packages.

so any problem ?

**Comment by SigireddyBalasai** [comment]
if possible include full context your os the version of python you are using and the error message

**Comment by BlackPool888** [comment]
I also cannot output image content,I can only get the main idea of the images,oops~

**Comment by BlackPool888** [comment]
Python:3.12
---------------
e.g:

![Image](https://github.com/user-attachments/assets/d0f80244-3215-4425-9852-a0f07dc42128)

**Comment by SigireddyBalasai** [comment]
@BlackPool888 i think you are getting what is intended for giving image input you will get description

**Comment by BlackPool888** [comment]
@SigireddyBalasai Right,I get it now,I had expected to receive the main content,not just a description,thx

**Comment by happy-xlf** [comment]
md = MarkItDown(llm_client=client, llm_model="gpt-4o") ,图片转md需要提供llm的接口才能识别

**Comment by neverlatetolearn0** [comment]
> [@SigireddyBalasai](https://github.com/SigireddyBalasai) Right,I get it now,I had expected to receive the main content,not just a description,thx

This may require an OCR approach

**Comment by neverlatetolearn0** [comment]
> md = MarkItDown(llm_client=client, llm_model="gpt-4o") ,图片转md需要提供llm的接口才能识别

This method can summarize the content of the picture, but can not identify the text in the picture, how to extract the text in the picture

**Comment by wking2014** [comment]
Insert the pictures in the PDF document into the corresponding positions in the Markdown document. 
This cannot be achieved.

**Comment by zenoda** [comment]
> > md = MarkItDown(llm_client=client, llm_model="gpt-4o") ,图片转md需要提供llm的接口才能识别
> 
> This method can summarize the content of the picture, but can not identify the text in the picture, how to extract the text in the picture

I also need this feature!
